plugin_ide.ui/src-lang/melnorme/lang/ide/ui/editor/structure/EditorStructureUtil.java

```diff
@@ -11,28 +11,29 @@
 package melnorme.lang.ide.ui.editor.structure;
 
 import static melnorme.lang.ide.ui.EditorSettings_Actual.EDITOR_ID;
-import melnorme.lang.ide.ui.editor.EditorUtils;

```

Comment:
Eclipse rearranged those imports for me. I hope, you don't mind. From my point of view the change makes sense as `org` before `melnorme` seems to be the dominant package order in the code base. Please correct me if I'm wrong.